io.trustable.stpadocumentationtool.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "io.trustable.stpadocumentationtool",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "stpa_documentation_tool",
+    "rename-icon": "stpa_documentation_tool",
+    "rename-desktop-file": "stpa_documentation_tool.desktop",
+    "rename-appdata-file": "stpa_documentation_tool.appdata.xml",
+    "finish-args": [ "--socket=x11",
+                     "--filesystem=xdg-download" ],
+    "modules": [
+        {
+            "name": "yaml-cpp",
+            "buildsystem": "cmake",
```

Comment:
Theoretically `cmake-ninja` should offer faster builds.

Reply:
Done